torch/quantization/observer.py

```diff
@@ -592,11 +580,6 @@ def _forward(self, x_orig):
     def calculate_qparams(self):
```

Comment:
Do you need to call `torch.jit.export` on this method now?

Reply:
yeah, I think so, will add it